backend/settings/settings.py

```diff
@@ -419,6 +419,7 @@
 RECOMMENDATIONS_MIN_TRUST_SCORES = 1.5
 
 UPDATE_MEHESTAN_SCORES_ON_COMPARISON = False
+MEHESTAN_KEEP_N_FREE_CPU = server_settings.get("MEHESTAN_KEEP_N_FREE_CPU", 3)
```

Comment:
Maybe we could try with 2 first ?
```suggestion
MEHESTAN_KEEP_N_FREE_CPU = server_settings.get("MEHESTAN_KEEP_N_FREE_CPU", 2)
```